test/CalculatorGTest.cpp

```diff
@@ -0,0 +1,35 @@
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   CalculatorGtest.cpp
+ *  @date   Wed May 26 2021
+ *  @brief  Calculator Unit Tests
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include "../inc/Calculator.h"
+
+using namespace testing;
+
+//note formatCalculation does no calculation, it simply puts the numbers and char into a string
+TEST(formatCalculationTest, BasicFormat)
```

Comment:
Each unit test should have a small description of what it does. An example is:
/**
 * @brief Unit test for the GET "application/v1/applicationAccessPoints/{applicationAccessPointId}" end point, get is called
 *        with valid access point data, ensures all elements are correct and OK response is returned
 *
 */